js/src/components/paid-ads/asset-group/asset-group-section.js

```diff
@@ -39,12 +40,23 @@ export default function AssetGroupSection() {
 				}
 			) }
 			description={
-				<p className="gla-asset-group-section__primary-description">
-					{ __(
-						'Create ads that effectively boost visibility and generate maximum conversions. Google will mix and match assets to create optimized ads in a variety of formats— maximizing your campaign’s performance.',
-						'google-listings-and-ads'
-					) }
-				</p>
+				<>
+					<p className="gla-asset-group-section__primary-description">
+						{ __(
+							'Upload text and image assets to effectively reach and engage your target shoppers. Google will mix and match your assets, continually testing combinations to create personalized and optimal shopping experiences.',
+							'google-listings-and-ads'
+						) }
+					</p>
+					<p>
+						<AppDocumentationLink
+							context="asset-groups"
+							linkId="asset-groups-learn-more"
```

Comment:
As the meaning of "asset group" on this page is a single data collection of grouped assets, it would be a bit properer to use a singular noun.

Reply:
Done https://github.com/woocommerce/google-listings-and-ads/pull/2009/commits/3546fdf71d250cca6e15a267a654843a4842efc5